cdap-ui/app/cdap/components/Cloud/Profiles/Preview/index.js

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {MyProfileApi} from 'api/cloud';
+import {getCurrentNamespace} from 'services/NamespaceStore';
+import LoadingSVG from 'components/LoadingSVG';
+import IconSVG from 'components/IconSVG';
+require('./Preview.scss');
+
+export default class ProfilePreview extends Component {
+  static propTypes = {
+    profileName: PropTypes.string,
+    profileCustomProperties: PropTypes.object
```

Comment:
This prop is not used yet?

Reply:
Yes. The next step is customizing a profile while attaching it to a manual run or schedule. At that time those customizations will be passed on here to be rendered. This and error will be used as the feature inches towards completion. This is just the base work.